src/hardware/vga.cpp

```diff
@@ -332,11 +332,29 @@ void VGA_SetCGA4Table(uint8_t val0,uint8_t val1,uint8_t val2,uint8_t val3) {
 
 void VGA_SetVga200LineHandling(const Vga200LineHandling vga_200_line_handling)
 {
+	if (vga.draw.vga_200_line_handling == Vga200LineHandling::ForceSingleScan) {
```

Comment:
It's tricky because in the VGA docos the description of the double scan bit only talks about 200-line modes. Which is accurate as only the 200-line standard modes are double scanned.

So the name `vga.draw.vga_200_line_handling` has to stay.

But we know games and demos use all sort of custom resolutions having 199, 240, 300 lines, etc. and all sub-350 lines must be double scanned (otherwise 31kHz VGA monitors could not display them).

Reply:
Nevermind, I got confused; this is all our code... Using "sub350" instead of "200" everywhere will be the best approach, as discussed.

Reply:
Updated